chisel/src/main.rs

```diff
@@ -224,6 +224,14 @@ fn execute_module(context: &ModuleContext, module: &mut Module) -> bool {
                 Err("trimexports: Invalid preset")
             }
         }
+        "trimstartfunc" => {
+            if let Some(start_section) = module.start_section() {
```

Comment:
Can you please create a module for this? See #88 for an example of a tiny module.

There are two reasons:
- chisel is only a frontend, perhaps others want to use libchisel directly
- we can add test cases in libchisel

Reply:
@axic thanks for the comments, moved to a new module and added some test cases.

Reply:
pending to add documentation, still not sure about how it should be configured in chisel.yml because this should work the same for "ewasm" or "pwasm".